internal/slack_integration/slack.go

```diff
@@ -77,7 +77,16 @@ func (b *Integration) handleEventAPI(ctx context.Context, event slackevents.Even
 		switch ev := event.InnerEvent.Data.(type) {
 		case *slackevents.MessageEvent:
 			if ev.ThreadTimeStamp == "" {
-				err := b.bot.Notify(ctx, ev.Channel)
+				// Get channel info before notifying
+				channelInfo, err := b.client.Client.GetConversationInfo(&slack.GetConversationInfoInput{
+					ChannelID: ev.Channel,
+				})
+				if err != nil {
+					log.Printf("Error getting channel info: %v", err)
+					return
+				}
+
+				err = b.bot.Notify(ctx, ev.Channel, channelInfo.Name)
```

Comment:
oh we should not do it everytime we get a message. probably add a channel without name and fill it async?